packages/maps/src/components/result/ReactiveMap.js

```diff
@@ -89,7 +90,7 @@ class ReactiveMap extends Component {
 		};
 
 		this.internalComponent = `${props.componentId}__internal`;
-		props.setQueryListener(props.componentId, props.onQueryChange, null);
+		props.setQueryListener(props.componentId, props.onQueryChange, props.onError);
```

Comment:
Here we can call `setError` and set the error for this component in redux store

Reply:
Didn't get you, we're already maintaining error state in redux.
`onError` is a callback which gets triggered when there is an error in `_msearch` request.

Reply:
Okay, but for maps we might have error other than elasticsearch error, which we can add in redux store, WDYT?

Reply:
Right but it's not related to this line of code.